playground/mocks/authenticator/challenge.js

```diff
@@ -1,5 +1,5 @@
 // To use this mock,
-// in terminal, run `yarn authenticator` to start the authenticator mock server
+// in terminal, run `yarn mock:authenticator` to start the authenticator mock server
```

Comment:
nit:
maybe remove the comment from those mock files and add to `playground/mocks/authenticator/README.md`